src/Symfony/Bundle/FrameworkBundle/Templating/Loader/TemplateLocator.php

```diff
@@ -36,6 +37,7 @@ public function __construct(FileLocatorInterface $locator, $cacheDir = null)
             $this->cache = require $cache;
         }
 
+        $this->rootDir = realpath(__dir__.'/../../../../../../../../../');
```

Comment:
This is totally wrong. We shouldn't rely on the location Symfony is installed in.